composer.json

```diff
@@ -9,6 +9,7 @@
         "php": ">=7.1",
         "ext-rdkafka": "*",
         "ext-json": "*",
+        "ext-pcntl": "*",
```

Comment:
If High Performance is optional, the extension should not be required by everyone.
Perhaps the existence of it could be checked on AbstractConnector::setHighPerformance and an exception could be thrown in case of absence

Reply:
I was thinking that high performance, besides its optionality, it would come as default, so the extension would have to exists if the user doesn't make any explicit opt-out in the config.